components/molecule/select/src/index.js

```diff
@@ -13,6 +13,7 @@ import {getCurrentElementFocused} from '@s-ui/js/lib/dom'
 
 const BASE_CLASS = `sui-MoleculeSelect`
 const CLASS_FOCUS = `${BASE_CLASS}--focus`
+const CLASS_DISABLED = `${BASE_CLASS}--disabled`
```

Comment:
Disabled satet should be treated as state classname following the SUIT conventions: `.is-disabled`

Reply:
DONE!